Makefile

```diff
@@ -112,6 +102,34 @@ ifeq ($(TARGET_ARCH),arm64)
 endif
 endif
 
+ifeq ($(TARGET_OS),windows)
+ifeq ($(SOURCE_OS),linux)
+    ifeq ($(shell which x86_64-w64-mingw32-gcc > /dev/null; echo $$?), 1)
+        $(info MinGW cross compiler not found, installing...)
+        $(shell sudo apt-get update && sudo apt-get install -y mingw-w64)
+    endif
+endif
+ifeq ($(TARGET_ARCH),amd64)
+    GO_TAGS ?= -tags no_cgo
+    # We need the go build command to think it's in cgo mode
+    export CGO_ENABLED = 1
+    export CXXFLAGS := -pthread
+    export CGO_CXXFLAGS := -pthread
```

Comment:
why do we need CPP flags?

Reply:
Removed these pthread tags.

Somewhere along the line FFmpeg (or its peripherals) involves cpp.

If I do not link the standard cpp library, we see `DLL Name: libstdc++-6.dll` refs in the executable.